pkg/controllers/membercluster/membercluster_controller.go

```diff
@@ -47,6 +47,11 @@ const (
 	reasonMemberClusterJoined         = "MemberClusterJoined"
 	reasonMemberClusterLeft           = "MemberClusterLeft"
 	reasonMemberClusterUnknown        = "MemberClusterUnknown"
+
+	fleetLabelKey              = "fleet-resource"
```

Comment:
We now have a fleet wide "prefix" https://github.com/ryanzhang-oss/fleet/blob/9434986b703d8f45ae601957629e96e9b624495c/apis/placement/v1beta1/commons.go
https://github.com/Azure/fleet/blob/9434986b703d8f45ae601957629e96e9b624495c/apis/placement/v1beta1/commons.go#L19